addOns/commonlib/src/main/javahelp/help/contents/alerttags.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<html>
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <title>
+        Alert Tags
+    </title>
+</head>
+<body>
+<h1>Alert Tags</h1>
+
+The Common Library add-on provides Alert Tags for use by any or all scan rules.
+
+<p>
+Of note the following tags/groups of tags are included:
+<ul>
+    <li>Custom Payloads &rarr; A tag which indicates the scan rules which support <a href="https://www.zaproxy.org/docs/desktop/addons/custom-payloads/">Custom Payloads functionality</a>.</li>
```

Comment:
Why arrows and why HTML entities? The file can use UTF-8.

Reply:
I originally had greater than but then I figured arrows looked more purposeful. I went with HTML entities because I figured that was safest for web site conversion etc.

Happy to use something else, just need input. I guess colons could be fine 🤷‍♂️